dataconverters.py

```diff
@@ -68,17 +66,17 @@ def convertMMCToXarrayNCDF(pathbase,year,dataDir,ncDir):
             if (not fname.endswith('.dat')) and (not fname.endswith('.txt')):
                 raise IOError('input file has neither .dat nor .txt extension. Bailing out!!\n')
             else:
-                name,_ = os.path.splitext(fpath)
+                name,fileExtension = os.path.splitext(fpath)
+                print(fileExtension)
             if(name+".nc" in outDirContents):
                 print("MMC file-- "+fname+" already xarray/netcdf-ized!\n")
             else:
-                fptr = open(fpath,'r')
-                db = read_mmc_database(fptr)
-                fptr.close()
+                db = MMCData(asciifile=fpath, **kwargs)
                 #Now make an xarrays object out of the db-dictionary/database
-                xrDS = dbToXarray(db)
-                outfilename ="{:s}{:s}.nc".format(outpath,fname.split(fileExtension)[0])
-                xrDS.to_netcdf(outfilename,unlimited_dims='Times')
+                xrDS = db.to_xarray()
+                outfilename ="{:s}/{:s}.nc".format(outpath,fname.split(fileExtension)[0])
+                print(outfilename)
+                xrDS.to_netcdf(outfilename,unlimited_dims='Times',encoding={'datetime':{'units': 'seconds since 1970-01-01 00:00:00.0'}})
```

Comment:
for readability, would recommend breaking up larger lines like this into something like:
```
xrDS.to_netcdf(outfilename,
               unlimited_dims='Times',
               encoding={
                   'datetime': {'units': 'seconds since 1970-01-01 00:00:00.0'}
               })
```
or whatever looks good to you.